server/mailet/integration-testing/pom.xml

```diff
@@ -194,6 +199,11 @@
                   <scope>test</scope>
               </dependency>
               <dependency>
+                  <groupId>com.jayway.jsonpath</groupId>
```

Comment:
JsonPath allow some semantic like : 

```
        assertThat(
            ImmutableList.of(jsonPath1.read("uid"),
                jsonPath2.read("uid"),
                jsonPath3.read("uid")))
            .containsOnly(ICS_UID,
                "f1514f44bf39311568d640727cff54e819573448d09d2e5677987ff29caa01a9e047feb2aab16e43439a608f28671ab7c10e754ce92be513f8e04ae9ff15e65a9819cf285a6962bd",
                "f1514f44bf39311568d640727cff54e819573448d09d2e5677987ff29caa01a9e047feb2aab16e43439a608f28671ab7c10e754ce92be513f8e04ae9ff15e65a9819cf285a6962be");
```

Wich is not easily possible with json-unit. Let's keep like this.